portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -25,7 +25,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String urlToRedirectToAfterUserLogsOut = "/";
       String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
 
-      response.getWriter().println(isLoggedIn);
       response.getWriter().println("<p>Hello " + userEmail + "!</p>");
       response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
     } else {
```

Comment:
Is the "urlToRedirectToAfterUserLogsIn" variable used just once below? If so, worth removing the variable altogether. If you deem that as a constant that will be used elsewhere, make it a class member variable.